cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/ConnectionHandler.java

```diff
@@ -478,7 +478,6 @@ private Connector getConnector(ServicePluginConfigurer configurer, PluginInfo pl
     MacroEvaluator macroEvaluator = new DefaultMacroEvaluator(new BasicArguments(arguments), evaluators,
                                                               Collections.singleton(OAuthMacroEvaluator.FUNCTION_NAME));
     MacroParserOptions options = MacroParserOptions.builder()
-                                 .skipInvalidMacros()
```

Comment:
I think the answer is no, but do we need to worry about anything compatibility related here?

Reply:
Don't think we need to worry about compatibility, though this is behavior change, but in 6.5, skipping this macro will make the endpoint fail at a later point, and provide a much more confusing error message to the user